tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932240.yaml

```
@@ -335,3 +335,20 @@ tests:
         output:
           log:
             expect_ids: [932240]
+  - test_id: 20
+    desc: "False positive against Googles GS2 cookie format"
+    stages:
+      - input:
+          dest_addr: "127.0.0.1"
+          method: "POST"
+          port: 80
+          headers:
+            User-Agent: "OWASP CRS test agent"
+            Host: "localhost"
+            Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            Cookies: _ga=GA1.1.859495864.1741097619; _ga_SM37QL5J42=GS2.1.s1747402925$o58$g1$t1747403211$j0$l0$h0
```

This typo is why your test is passing.

```suggestion
            Cookie: _ga=GA1.1.859495864.1741097619; _ga_SM37QL5J42=GS2.1.s1747402925$o58$g1$t1747403211$j0$l0$h0
```

---

ouch. Nice catch.